src/content/docs/usage/configuration.md

```diff
@@ -92,6 +92,10 @@ For Nextflow DSL2 nf-core pipelines - parameters defined in the parameter block
 You can also generate such a JSON via each pipelines 'launch' button on the [nf-co.re website](https://nf-co.re/launch).
 :::
 
+:::warning
```

Comment:
I don't think that we need this one? It hasn't been introduced at this point. I'd rather not duplicate the warning, maybe just have the one below...

Reply:
The issue is we split the how to write a custom config with the tuning bit.

Depending on how the person entered the page, they would miss the warning if it was only in one place...

I'm not actually very happy with the structure of the page in regards to highest priority info but didn't have the time to restructure it :/ that was the simplest I could do...

Reply:
Actually changed my mind, worked out a way to do it properly - will update